packages/cli/scripts/seeder.ts

```diff
@@ -62,10 +62,10 @@ const main = async () => {
   const peer3 = Client.http({ port: args.rpcPort + 3 })
   const clients = [ultralight, peer1, peer2, peer3]
   const clientInfo: Record<Clients, TestClient> = {
-    ultralight: { client: ultralight, enr: '', nodeId: '' },
-    peer1: { client: peer1, enr: '', nodeId: '' },
-    peer2: { client: peer2, enr: '', nodeId: '' },
-    peer3: { client: peer3, enr: '', nodeId: '' },
+    ultralight: { client: ultralight, enr: 'enr:-IS4QFV_wTNknw7qiCGAbHf6LxB-xPQCktyrCEZX-b-7PikMOIKkBg-frHRBkfwhI3XaYo_T-HxBYmOOQGNwThkBBHYDgmlkgnY0gmlwhKRc9_OJc2VjcDI1NmsxoQKHPt5CQ0D66ueTtSUqwGjfhscU_LiwS28QvJ0GgJFd-YN1ZHCCE4k', nodeId: '' },
```

Comment:
Please revert these changes.  This object is populated in the course of running this script - see lines 130-137 - and this causes the CI to fail.

Reply:
You had a bug in the updated enr validator in the RPC.  I just pushed a commit to fix.  CI should pass now so will merge once it does